piptools/_compat/__init__.py

```diff
@@ -1,3 +1,4 @@
+from .click_compat import IS_CLICK_VER_8_PLUS
 from .pip_compat import PIP_VERSION, parse_requirements
 
-__all__ = ["PIP_VERSION", "parse_requirements"]
+__all__ = ["PIP_VERSION", "IS_CLICK_VER_8_PLUS", "parse_requirements"]
```

Comment:
Side note: why is this a list? It's usually recommended to make `__all__` immutable, hence use a tuple.

P.S. This is out of the scope of this PR, just wanted to make a note.

Reply:
to be fair, this is a list as well in the python tutorial on modules: https://docs.python.org/3/tutorial/modules.html#importing-from-a-package and the pip package has this format as well. 
Do you want to change this here or should I leave as it is? I actually don't have any preference here.

Reply:
No, don't touch this because it's out of the scope. But in the future, this should probably be added. I've added this comment for history.

P.S. Normally, pylint would warn about this problem but I don't think it's integrated.

Reply:
at least pycodestyle seems to do so, compare https://github.com/PyCQA/pydocstyle/issues/62